app/models/version.rb

```diff
@@ -0,0 +1,62 @@
+class Version < ActiveRecord::Base
+  belongs_to :target, polymorphic: true
+
+  validate :numbers_must_increase
+  validate :draft_cannot_be_behind_live
+  validate :live_is_equal_to_draft
```

Comment:
:+1: I think this is a good spot.

It's perfectly valid for the Version for live item to differ from the draft. This invariant should only really apply at the point of publish. Maybe this is something we move into the publish command or conditionalise it on the model.

What do people think?